core/src/com/unciv/logic/map/MapParameters.kt

```diff
@@ -128,15 +128,15 @@ object MapShape : IsPartOfGameInfoSerialization {
 }
 
 object MapType : IsPartOfGameInfoSerialization {
+    const val default = "Default"
     const val pangaea = "Pangaea"
     const val continents = "Continents"
     const val fourCorners = "Four Corners"
-    const val perlin = "Perlin"
     const val archipelago = "Archipelago"
     const val innerSea = "Inner Sea"
 
     // Cellular automata
-    const val default = "Default"
+    const val cellularAutomata = "Cellular Automata"
```

Comment:
Please call this something that users can understand - 'cellular automata' isn't a map type, is a map generation type, and users don't know what to expect